### PR TITLE
cfg.usb.cfg:modify cfg to analyze the usb_multi_disk case

### PR DESCRIPTION
**Pull request by maxujun**
cfg.usb.cfg:modify cfg to analyze the usb_multi_disk case
ID:1411609
Signed-off-by: Xujun Ma <xuma@redhat.com>

### CODE REVIEW

**Comment by CongLi** [comment]
@maxujun 

Could you please also update line 174 which is the same problem of it?

Thanks.

**Comment by maxujun** [comment]
@CongLi  Have already updated it.

File: qemu/tests/cfg/usb.cfg [hunk 1]
```diff
@@ -262,7 +262,7 @@
                 - usb_multi_times:
                     usb_repeat_times = 5000
         - usb_multi_disk:
-            only ehci
+            only usb-ehci
```
Comment (CongLi) [inline qemu/tests/cfg/usb.cfg]: One more question:
As you said, usb-ehci controller can not connect with low-speed usb device when machine type is rhel6.*.

how to solve it on rhel6 host?

Comment (maxujun) [inline qemu/tests/cfg/usb.cfg]: The usb-storage is not a low-speed  device in usb_multi_disk case,so it can be attached to usb-ehci.

**Comment by qunfang** [comment]
Will this make many cases skipped on ppc after BZ1410674 fixed?

File: qemu/tests/cfg/usb.cfg [hunk 2]
```diff
@@ -171,7 +171,7 @@
             variants:
                 - usb_normal_test:
                 - usb_negative_test:
-                    only ehci
+                    only usb-ehci
                     only usb_hub
```
Comment (MiriamDeng) [inline qemu/tests/cfg/usb.cfg]: It seems that the related test case can not be filtered here.

**Comment by CongLi** [comment]
Close it as @maxujun has no time to handle it.

**Pull request closed without merging**